### DISCUSSION TITLE
How to prevent one channel of powerstrip from switch off?

### DISCUSSION THREAD

**Discussion by guny74** [arendst/Tasmota#11309] [Q&A] [answered]
Hello,

I have a 5 channel powerstrip with tasmota 9.2 One of the channels are USB plugs and I want to prevent only this  Channel to be switched off.

The are a lot of cammand in the documentation, but I di not find the right one.

Is this possible and if how ?

Thank you

Regards

**Comment by sfromis** [comment, marked as answer]
One way would to to configure the relay pin as `Output Hi`, meaning that it will no longer appear as a relay, and always be on as long as Tasmota is in control. Usually, `PowerOnState` is the way to go, but this feature cannot be restricted to one of the relay outputs.

**Reply by guny74** [reply]
Hello,

thank, but how/where can I configure the relay pin an "Output Hi" ?

**Reply by guny74** [reply]
Hello,

I found it in the template. Thank you
